docs/developer/base/basics.md

```diff
@@ -1,3 +1,165 @@
 # Basics
 
 -----
+
+The AgentCheck base class in base.py contains the logic that all Checks inherit.
+
+In addition to the integrations inheriting from AgentCheck, other classes that inherit from AgentCheck include:
+
+- [PDHBaseCheck](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/win/winpdh_base.py)
+- [OpenMetricsBaseCheck](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/openmetrics/base_check.py)
+- [GenericPrometheusCheck](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/prometheus/base_check.py)
+- [PrometheusCheck](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/prometheus/prometheus_base.py)
+- [KubeLeaderElectionBaseCheck](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/kube_leader/base_check.py)
+
+
+## Getting Started
+In each Check, Datadog looks for `__version__` and a subclass of AgentCheck at the root of every package.
+
+Below is an example of the `__init__.py` folder for AwesomeCheck:
```

Comment:
I'm gonna be super annoying here, but naming something `AwesomeCheck` as an example isn't in the dispassionate tone we usually like to strike in the docs. Let's change this to `MyCheck` or something equally dry.

Reply:
Yes, though these are dev docs and are not on the corp site, so it's okay. We also use lots of GIFs 😄